library/L1_Peripheral/msp432p401r/system_controller.hpp

```diff
@@ -72,11 +72,36 @@ class SystemController final : public sjsu::SystemController
     kSystem,
   };
 
+  /// The available clock dividers for the primary clocks.
+  enum ClockDivider : uint8_t
+  {
+    //! @cond Doxygen_Suppress
+    kDivideBy1   = 0b000,
+    kDivideBy2   = 0b001,
+    kDivideBy4   = 0b010,
+    kDivideBy8   = 0b011,
+    kDivideBy16  = 0b100,
+    kDivideBy32  = 0b101,
+    kDivideBy64  = 0b110,
+    kDivideBy128 = 0b111,
+    //! @endcond
+  };
+
+  class Peripherals
```

Comment:
I was certain I was going to hate the name Peripherals for this and now I see why. What you did is the correct thing, but now we need to come up with a better semantic word for what these things are. Do you have any ideas?

Reply:
What about naming it as Modules?

Reply:
Yeah, you can rename it to that. Need to figure out a good name for the PeripheralID variable.